### ISSUE TITLE
define fields for auto-generation

### ISSUE THREAD

**Issue by mrderiuzz** [lucwastiaux/anki-language-tools#8]
can we get a seperate view to define all fields that we'd like to have audio auto generated for? 

ideally something like language mapping, so originally empty, then we could define: deck, source field, and target field. I was originally very confused that I first need to do it from "Browse" view and then it's stored.

Also this works on deck level but subdecks are not included, I was confused but then I got it (wasn't clear to me). Perhaps on this "config" page we could also define if we want to include subdecks or not, that'd be pretty great

**Comment by lucwastiaux** [comment]
Does **Language Tools -> Show Settings for Selected Notes** answer your question ? I agree the doc needs to be better. Let's use this issue for a discussion on how it should be.

**Comment by mrderiuzz** [comment]
Yeah I learned that's it's available in "show settings for notes" but it's
totally not untivite it's there. What do you think about an aggregated view
for all notes? Something like language mapping but for audio generation
settings (with option to add new settings there too)
